ext/opentelemetry-ext-elasticsearch/tests/helpers_es2.py

```diff
@@ -0,0 +1,33 @@
+from elasticsearch_dsl import (  # pylint: disable=no-name-in-module
+    DocType,
+    String,
+)
+
+
+class Article(DocType):
+    title = String(analyzer="snowball", fields={"raw": String()})
+    body = String(analyzer="snowball")
+
+    class Meta:
+        index = "test-index"
+
+
```

Comment:
`helper_esN.py` files are a bit repetitive. They could import each other or import from a common base file and modify the data but that would introduce logic and I think it's better to treat these as data/declarations. I think some duplication is better than adding logic in such places. 

Also: https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html

Reply:
Is this the reason for multiple test environments? Why does each case need to be in a seperate env?

Reply:
We are testing multiple versions of elasticsearch lib. Each version is a separate packge and has it's own dependencies so tox envs seem to be the ideal way of handling this. Are there other ways this can be done?